lib/index.js

```diff
@@ -317,15 +333,13 @@ internals.getParamsData = function (param, name) {
             });
         }
 
-        ['includes', 'excludes', 'items'].forEach(function (rule) {
-
-            if (param[rule]) {
-                data.rules[internals.capitalize(rule)] = param[rule].map(function (ruleType) {
-
-                    return internals.getParamsData(ruleType);
-                });
-            }
-        });
+        // If we have only one specific rule then set that to our type for
```

Comment:
This looks a bit weird, replacing the type with a rule, in which use case does it happen ?